sympy/stats/tests/test_joint_rv.py

```diff
@@ -236,10 +236,20 @@ def test_expectation():
     m = Normal('A', [x, y], [[1, 0], [0, 1]])
     assert simplify(E(m[1])) == y
 
-@XFAIL
+
 def test_joint_vector_expectation():
     m = Normal('A', [x, y], [[1, 0], [0, 1]])
-    assert E(m) == (x, y)
+    assert E(m).simplify() == Matrix([[x], [y]])
```

Comment:
what happens without `simplify()`?

Reply:
This results were a bit hard to verfiy without `simplify`:
```py
>>> m = Normal('A', [x, y], [[1, 0], [0, 1]])
>>> E(m)
Matrix([
[
    x*(2 - erfc(sqrt(2)*y/2))/2 + x*erfc(sqrt(2)*y/2)/2],
[-y*(-2*sqrt(pi)*(2 - erfc(sqrt(2)*y/2)) - 2*sqrt(2)*exp(-y**2/2)/y)/(4*sqrt(pi)) - y*(-2*sqrt(pi)*erf
c(sqrt(2)*y/2) + 2*sqrt(2)*exp(-y**2/2)/y)/(4*sqrt(pi))]])
```

Reply:
The univariate normal distribution works without `.simplify()`. Can you check why?

Reply:
> The univariate normal distribution works without `.simplify()`.

That's because for univariate RVs, one the way is to calculate using moment generating function. If we comment out the following lines:
https://github.com/sympy/sympy/blob/8f6bc5947262754fe623d30682e47b7e01580bf7/sympy/stats/crv_types.py#L2857-L2871

The following will be affected and return from line 380:
https://github.com/sympy/sympy/blob/8f6bc5947262754fe623d30682e47b7e01580bf7/sympy/stats/crv.py#L378-L380

Therefore if we comment out the above lines we get the following results:
```py
>>> x = symbols('x')
>>> s1 = symbols('sigma1', positive=True)
>>> N = Normal('N', x, s1)
>>> E(N)
-x*(-2*sqrt(2)*sigma1*exp(-x**2/(2*sigma1**2))/x - 2*sqrt(pi)*(2 - erfc(sqrt(2)*x/(2*sigma1))))/(4*sqrt(pi)) - x*(2*sqrt(2)*sigma1*exp(-x**2/(2*sigma1**2))/x - 2*sqrt(pi)*erfc(sqrt(2)*x/(2*sigma1)))/(4*sqrt(pi))
>>> E(N).simplify()
x
```

Reply:
Do you think we can port the moment-generating function approach to multivariate RVs?

Reply:
> moment-generating function approach to multivariate RVs?

I need to find some readings related to this topic

Reply:
OK, no problem. Let's focus on fixing and testing the code first.